README.md

```diff
@@ -1,2 +1,57 @@
+
+### Custom content
+Add any view to the center of a circular progress bar as a content to: indicate the completion of a task or process; add start, pause, or cancel buttons to control the progress interactively; add an image that indicates the actual task in progress; and add custom text that conveys the progress unit.
+
+### Visualize multiple ranges
+Along with visualizing the progression of a task, users can also visualize multiple ranges with gradient colors or solid colors based on the range to enhance readability.
+
+## Related Links
+
+[Learn More about .NET MAUI ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[Documentation](https://help.syncfusion.com/maui/circularprogressbar/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[View Demos](https://github.com/syncfusion/maui-demos/tree/master/MAUI/ProgressBar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[Community Forums](https://www.syncfusion.com/forums/maui?control=sfcircularprogressbar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples)
+
+[Knowledge Base](https://support.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples)
+
+## About Syncfusion .NET MAUI Controls
+
+Syncfusion's [.NET MAUI UI Controls](https://www.syncfusion.com/maui-controls/maui-progressbar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to ProgressBar, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples), and [Excel Library](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples).
```

Comment:
Recheck on this
![Uploading image.png…]()

Reply:
Corrected